pkg/ccl/backupccl/show.go

```diff
@@ -222,6 +233,41 @@ func backupShowerDefault(ctx context.Context, p sql.PlanHookState, showSchemas b
 	}
 }
 
+func showPrivileges(descriptor sqlbase.Descriptor) string {
```

Comment:
Is there a SQL function that can pretty print privileges (maybe even as `GRANT`s? I wonder if we can reuse the logic from `dump`?